_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,254 @@
+name change has brought clarity to the community. Trino is an improved version,
+supported by the founders and creators of Presto®,  along with the major
+contributors.
+
+And if you have not done so already, make sure to [star the
+repository]({{site.github_repo_url}}) and [join us on slack](/slack.html).
+
+## Community resources and events
+
+Beyond the raw code and helping each other, the community was also busy
+collaborating on other media.
+
+The book [Presto: The Definitive Guide](/presto-the-definitive-guide.html) was
+completed by Matt, Manfred and Martin and published by O'Reilly. And over 5000
+readers took advantage of the [free digital copy]({% post_url
+2020-04-11-the-definitive-guide %}).
+
+Brian and Manfred launched the live streaming event [Trino Community
+Broadcast](broadcast/index.html), and grew their audience and back catalog to
+include some very useful material. If you have not seen it yet, go and [watch
+some old episodes](/broadcast/episodes.html) and join us in the next ones.
+
+We also had a number of other online events and presentations, with direct
+participation of our community members:
+
+* A [dedicated conference event](% post_url 2020-11-21-a-report-about-presto-conference-tokyo-2020 %})
+  for the community in Japan was very successful.
+* The [Argentina Big Data Meetup](% post_url
+  2020-09-28-argentina-big-data-meetup %}) had a large audience from the
+  community in South America
+
+A series of virtual events around the project started with a roadmap and
+overview meeting and included a number real world use case examples at scale:
+
+* [State of Trino]({% post_url 2020-05-15-state-of-presto %})
+* [Trino at Pinterest]({% post_url 2020-07-22-presto-summit-pinterest %})
+* [Trino Migration at ARM Treasure Data]({% post_url 2020-07-06-presto-summit-arm-td %})
+* [Trino at Zuora]({% post_url 2020-06-16-presto-summit-zuora %})
+
+Another series of training classes with the project founders was hugely
+successful. It includes very valuable content for any Trino user, from beginners
+to experts, that you should not miss:
+
+* [Advanced SQL in Trino with David]({% post_url 2020-07-15-training-advanced-sql %})
+* [Understanding and Tuning Trino Query Processing with Martin]({% post_url 2020-07-30-training-query-tuning %})
+* [Securing Trino with Dain]({% post_url 2020-08-13-training-security %})
+* [Configuring and Tuning Trino with Dain]({% post_url 2020-08-27-training-performance %})
+
+## Features and code
+
+While everything mentioned is already exciting, the true work is visible in the
+new features and improvements in Trino. It is a long list, but read on. You
+won’t want to miss anything.
+
+### Improvements to ANSI SQL support
+
+A core feature of Trino is the ability to use the same standard SQL for any
+connected data source. These improvements empower all users.
+
+* Variable-precision temporal types, with precision down to picoseconds. This a
```

Comment:
as picoseconds are not in daily use for mortals, it can be helpful  to describe what 1 pico is